src/context/videos/Reducer.jsx

```diff
@@ -2,6 +2,16 @@ const videosReducer = (state, { type, payload }) => {
   switch (type) {
     case "FILTER_BY_CATEGORY":
       return { ...state, category: payload };
+    case "ADD_TO_LIKED":
+    case "REMOVE_FROM_LIKED":
```

Comment:
Nice move here. Not using break or default for the action types that perform the same thing. This way the developer knows what's happening without repeating the code for same thing.